packages/protocol/test/identity/odispayments.ts

```diff
@@ -0,0 +1,146 @@
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
+import {
+  assertEqualBN,
+  assertLogMatches2,
+  assertRevert,
+  assumeOwnership,
+} from '@celo/protocol/lib/test-utils'
+import { getDeployedProxiedContract } from '@celo/protocol/lib/web3-utils'
+import { fixed1 } from '@celo/utils/src/fixidity'
+import {
+  FreezerContract,
+  FreezerInstance,
+  OdisPaymentsContract,
+  OdisPaymentsInstance,
+  RegistryInstance,
+  StableTokenContract,
+  StableTokenInstance,
+} from 'types'
+
+const Freezer: FreezerContract = artifacts.require('Freezer')
+const OdisPayments: OdisPaymentsContract = artifacts.require('OdisPayments')
+const StableTokenCUSD: StableTokenContract = artifacts.require('StableToken')
+
+const SECONDS_IN_A_DAY = 60 * 60 * 24
+
+contract('OdisPayments', (accounts: string[]) => {
+  let freezer: FreezerInstance
+  let odisPayments: OdisPaymentsInstance
+  let registry: RegistryInstance
+  let stableTokenCUSD: StableTokenInstance
+
+  const owner = accounts[0]
+  const sender = accounts[1]
+  const startingBalanceCUSD = 1000
+
+  before(async () => {
+    // Mocking Registry.sol when using UsingRegistryV2.sol
+    registry = await getDeployedProxiedContract('Registry', artifacts)
+    if ((await registry.owner()) !== owner) {
+      // In CI we need to assume ownership, locally using quicktest we don't
```

Comment:
This is because in CI we run all migrations, including the one that transfers ownership to Governance, right?

Reply:
yes exactly!